plugins/common_issues/__init__.py

```diff
@@ -0,0 +1,321 @@
+"""CommonIssues plugin.
+
+| Copyright 2017-2023, Voxel51, Inc.
+| `voxel51.com <https://voxel51.com/>`_
+|
+"""
+
+import numpy as np
+from PIL import Image, ImageStat
+
+
+import fiftyone.operators as foo
+from fiftyone.operators import types
+import fiftyone as fo
+import fiftyone.core.utils as fou
+from fiftyone import ViewField as F
+
+
+def compute_sample_brightness(sample):
+    image = Image.open(sample.filepath)
```

Comment:
Yeah +1. We need to find a clean solution for writing plugins that will run in OSS and Teams with no code changes

Reply:
It's almost like we need common plugin utilities that do the right thing. Like what `six` module does/did for python2/3 compatibility.

Because you can imagine a future world where there's a plugin developer who is not a Teams customer so has no idea what `local_path` means. But we still want to encourage cross-compatibility.

Reply:
Now, you still can't force the plugin developer to use that utility, but it's a lot easier to reference that function in plugin development documentation instead of saying "Oh by the way if you want to open an image pretty *please* try to use `local_path` first, trust us ;) "

Reply:
Yeah exactly. I've been thinking to backport the `fiftyone.core.storage` from Teams to OSS for this reason. That entire module "magically" supports cloud paths in Teams, so if it existed in OSS (only supporting local paths), then we could encourage using it for all I/O related tasks.